test/junit/scala/collection/immutable/TreeMapTest.scala

```diff
@@ -10,6 +10,7 @@ import org.junit.runners.JUnit4
 import scala.tools.testkit.AllocationTest
 
 @RunWith(classOf[JUnit4])
+@annotation.nowarn("msg=Boolean literals should be passed using named argument syntax for parameter selfEqual")
```

Comment:
this isn't needed. It doesn't warn under old and won't warn under new. Are you paying attention?